composer.json

```diff
@@ -14,10 +14,13 @@
     }],
     "require": {
         "php": ">=5.3.0",
-        "symfony/framework-bundle" : ">=2.0"
+        "symfony/framework-bundle" : ">=2.0",
+        "doctrine/cache": "~1.3",
+        "guzzlehttp/guzzle": ">=4.0"
     },
     "require-dev": {
-        "twig/extensions": "1.0.*"
+        "twig/extensions": "1.0.*",
+        "phpunit/phpunit": "4.3.*"
```

Comment:
Yeah, dont add phpunit and also please use `^` or `~` instead of the `1.0.*` syntax when specifying dependencies.